### PR TITLE
feat(notifier): adds email channel

### PR DESCRIPTION
**Pull request by vjgene**
Fixes https://rsklabs.atlassian.net/browse/RMKT-789

### CODE REVIEW

**Comment by vjgene** [comment]
https://user-images.githubusercontent.com/11134695/123999455-d384fe00-d9d2-11eb-99c0-46e65208a632.mp4

**Review by itofarina — APPROVED** [review]
👌

File: src/utils/stringUtils.ts [hunk 1]
```diff
@@ -25,3 +25,5 @@ export const shortChecksumAddress = (address: string): string => shortenString(
 export const removeURLProtocol = (url: string) => url.replace(/^http[s]*:\/\//, '')
 
 export const trailingSlashRegex = new RegExp(/\/$/, '')
+
+export const emailRegex = new RegExp(/^\S+@\S+\.\S+$/)
```
Comment (0xartem) [inline src/utils/stringUtils.ts:29]: Alvaro said yesterday that we have this function built-in already, so I came to the conclusion that you shouldn't write your own regex, no?

**Review by 0xartem — APPROVED** [review]
overall looks good, have just one question

File: src/utils/stringUtils.ts (hunk 1 above)
Comment (vjgene) [reply src/utils/stringUtils.ts:29]: @artem-brazhnikov as we wanted to keep our check simple i used a regex. If the built in doesn't add lot of complexity like adding external modules, we can use it. @farinalvaro @jurajpiar any ideas where the built in function is?

Comment (itofarina) [reply src/utils/stringUtils.ts:29]: Hey, sorry, I was wrong yesterday. The way it's solved in the library examples is with a regex also.
https://react-hook-form.com/

**Review by jurajpiar — APPROVED** [review]

**Pull request merged**